stock_unassign_wizard/models/stock_move.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2021 PlanetaTIC - Marc Poch <mpoch@planetatic.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api, _
+
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+
+    @api.multi
+    def open_unassign_wizard(self):
+        self.ensure_one()
+        unassign_wizard_obj = self.env['stock.unassign.wizard']
+
+        confirmed_moves = self.search([
+            ('product_id', '=', self.product_id.id),
+            ('state', 'in', ['assigned', 'partially_available']),
+            '|',
+            '&',
+            '&',
+            ('picking_id', '!=', False),
+            ('picking_id', '!=', self.picking_id.id),
+            ('picking_type_id.code', 'in', ('outgoing', 'internal')),
+            '&',
+            ('raw_material_production_id', '!=', False),
```

Comment:
@MPoch-PlanetaTIC IMHO you should put the domain retrieval in a function in order to split this into two modules as pulling mrp constrains a little bit inheritance. What do you think ?

Reply:
You are right.
I'll split it in two functions:
* the main function: open_unassign_wizard
* and add the new function get_unassign_wizard_domain
Currently I have no time to do this fix, but I will do it.

Thanks